.github/workflows/checker-framework.yml

```diff
@@ -0,0 +1,35 @@
+name: Checker-Framework
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  checker-framework:
+    strategy:
+      matrix:
+        profile:
+          - checker-framework-nullness-optional-interning
+          - checker-framework-methods-resource-fenum
+          - checker-framework-lock-tainting
+          - checker-framework-index
+          - checker-framework-formatter
+          - checker-framework-signature-gui-units-init
+          - checker-framework-regex-property-key-compiler-message
+          - checker-framework-purity-value-returns
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 17
```

Comment:
Java 11 in CI was showing inconsistent behavior for `-Awarns`, but fine on my local. Trying Java 17 now to see if this helps.